dockerfiles/rinkeby/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '2.2'
+
+services:
+
+  vulcanizedb:
+    build:
+      context: ./../../
+      dockerfile: dockerfiles/rinkeby/Dockerfile
+    container_name: rinkeby_vulcanizedb
+    command: "sync --starting-block-number 0 --config /config.toml"
+    volumes:
+      - "./config.toml:/config.toml"
+      - "vulcanizedb_geth_data:/geth"
+    networks:
+      vulcanizedb_net:
+
+  migrations:
+    image: migrate/migrate:v3.3.0
+    container_name: rinkeby_vulcanizedb_migrations
+    depends_on:
+      postgres:
+        condition: service_healthy
+    command: -database postgresql://postgres:postgres@postgres:5432/vulcanizedb?sslmode=disable -path /migrations up
+    volumes:
+      - ./../../db/migrations:/migrations
+    networks:
+      vulcanizedb_net:
+
+  postgres:
```

Comment:
thanks for the PR @chapsuk! Would a volume or volume option make sense for the postgres container so the data is retained if the container is stopped?

Reply:
Awesome, thanks for the clarification.